items/plugin/appointment-manager/versions/NA.json

```
@@ -10,14 +10,15 @@
   },
   "itemId": "appointment-manager",
   "name": "Appointment Manager",
+  "description": "Manage appointments, availabilities, slots and exceptions.",
   "publishOnMiaDocumentation": true,
   "repositoryUrl": "https://git.tools.mia-platform.eu/mia-care/platform/plugins/appointment-manager",
   "resources": {
     "services": {
       "appointment-manager": {
         "type": "plugin",
         "name": "appointment-manager",
-        "dockerImage": "nexus.mia-platform.eu/plugins/appointment-manager:2.3.0",
+        "dockerImage": "nexus.mia-platform.eu/plugins/appointment-manager",
```

As a rule of thumb, `NA` versions are meant to be copies of the lowest "named" version, kept only for retrocompatibility
Removing the image tag would make it sort of a "latest" version, which is a concept versioned items should not have

Is there a particular reason why you need it to be this way?

---

No, we can revert it.
```suggestion
        "dockerImage": "nexus.mia-platform.eu/plugins/appointment-manager:2.3.0",
```